restcomm/configuration/config-scripts/as7-config-scripts/restcomm/autoconfig.d/config-restcomm.sh

```diff
@@ -67,16 +68,46 @@ configRestcomm() {
+                                -e "s|<external-ip>.*<\/external-ip>|<external-ip>$static_address<\/external-ip>|" \
+                                -e "s|<external-address>.*<\/external-address>|<external-address>$static_address<\/external-address>|" \
+                                -e "s|<\!--.*<external-address>.*<\/external-address>.*-->|<external-address>$static_address<\/external-address>|" \
+                                -e "s|<outbound-proxy-uri>.*<\/outbound-proxy-uri>|<outbound-proxy-uri>$outbound_proxy<\/outbound-proxy-uri>|" \
+                                -e "s|<outbound-proxy-user>.*<\/outbound-proxy-user>|<outbound-proxy-user>$outbound_proxy_user<\/outbound-proxy-user>|"  \
+                                -e "s|<outbound-proxy-password>.*<\/outbound-proxy-password>|<outbound-proxy-password>$outbound_proxy_password<\/outbound-proxy-password>|" $FILE > $FILE.bak;
+                        else
+                                # JH. URIs using HTTP. Add recordings-uri
+                                sed -e "s|<local-address>.*<\/local-address>|<local-address>$bind_address<\/local-address>|" \
+                                -e "s|<remote-address>.*<\/remote-address>|<remote-address>$bind_address<\/remote-address>|" \
+
+                                -e "s|<prompts-uri>.*<\/prompts-uri>|<prompts-uri>http://${static_address}:8080/restcomm/audio<\/prompts-uri>|" \
+                                -e "s|<cache-uri>.*<\/cache-uri>|<cache-uri>http://${static_address}:8080/restcomm/cache<\/cache-uri>|" \
+                                -e "s|<recordingss-uri>.*<\/recordings-uri>|<recordings-uri>http://${static_address}:8080/restcomm/recordings<\/recordings-uri>|" \
+
+                                -e "s|<\!--.*<external-ip>.*<\/external-ip>.*-->|<external-ip>$static_address<\/external-ip>|" \
+                                -e "s|<external-ip>.*<\/external-ip>|<external-ip>$static_address<\/external-ip>|" \
+                                -e "s|<external-address>.*<\/external-address>|<external-address>$static_address<\/external-address>|" \
+                                -e "s|<\!--.*<external-address>.*<\/external-address>.*-->|<external-address>$static_address<\/external-address>|" \
+                                -e "s|<outbound-proxy-uri>.*<\/outbound-proxy-uri>|<outbound-proxy-uri>$outbound_proxy<\/outbound-proxy-uri>|" \
+                                -e "s|<outbound-proxy-user>.*<\/outbound-proxy-user>|<outbound-proxy-user>$outbound_proxy_user<\/outbound-proxy-user>|"  \
+                                -e "s|<outbound-proxy-password>.*<\/outbound-proxy-password>|<outbound-proxy-password>$outbound_proxy_password<\/outbound-proxy-password>|" $FILE > $FILE.bak;
+                        fi
 		else
+                        # JH.  No static address; use relative URLs   ??? What's that mean??? On the same system, "file://"??  Does anything need to be done here?
```

Comment:
No need to do anything.
Remember we are covering a special case, where MS is installed in different server. In this case, both MS_ADDRESS and STATIC_ADDRESS must be specified.